diff --git a/ajax_pages/ajax_pages.js b/ajax_pages/ajax_pages.js
index a41de84..dc0ea44 100644
--- a/ajax_pages/ajax_pages.js
+++ b/ajax_pages/ajax_pages.js
@@ -260,6 +260,9 @@ Drupal.ajax.prototype.commands.ajaxPagesPageHtml = function(ajax, response, stat
       $element[0].scrollIntoView(true);
     }
   }
+  else {
+    $("html, body").animate({ scrollTop: 0 }, 0);
+  }

   // Attach all JavaScript behaviors to the new content, if it was successfully
   // added to the page, this if statement allows #ajax['wrapper'] to be
